lib/cardano-sl.cabal

```diff
@@ -139,6 +157,24 @@ library
                         Pos.Statistics.Ekg
                         Pos.Statistics.Statsd
 
+                        Pos.DHT.Workers
+
+                        Pos.Block.BlockWorkMode
```

Comment:
I suggest the following change to avoid this:

 `Pos.Block.Worker` depends on `Pos.Communication.Util`, which depends on `Pos.Diffusion.Types (Diffusion)`. If latter is moved elsewhere, `Pos.Block.Worker` will remain in `infra`. `Pos.Diffusion.Types` depends on different types from `block`, `ssc`, `infra`, but not on `lib`. It can be and must be moved elsewhere. I suggest `infra`, though you may want to create a separate package for it.

Reply:
To avoid what exactly? Is this a problem?

Reply:
Yes, we've applied a lot of effort to structure the code in terms of modules so that interactions between them are straightforward and easy to use. Moving a lot of logic from components to `lib` is a regress.

Reply:
Ok perhaps we can discuss this later, but surely you'll agree it's nowhere near as important and pressing as the substantive changes to the program.

Reply:
This change is trivial and I can do it in your branch if you want me to. Keeping code properly structured helps everybody in every single way, and it's also what reviews are for, so I'm just doing my job 🤷‍♂️